packages/headless/src/features/product-listing/product-listing-actions.ts

```diff
@@ -1,24 +1,27 @@
 import {ArrayValue, StringValue} from '@coveo/bueno';
 import {createAction, createAsyncThunk} from '@reduxjs/toolkit';
 import {getVisitorID} from '../../api/analytics/coveo-analytics-utils';
-import {AsyncThunkProductListingOptions} from '../../api/commerce/product-listings/product-listing-api-client';
 import {
-  ProductListingRequest,
-  ProductListingSuccessResponse,
+    AsyncThunkProductListingOptions,
+    AsyncThunkProductListingV2Options
+} from '../../api/commerce/product-listings/product-listing-api-client';
+import {
+    ProductListingRequest,
+    ProductListingSuccessResponse, ProductListingV2Request, ProductListingV2SuccessResponse,
 } from '../../api/commerce/product-listings/product-listing-request';
 import {isErrorResponse} from '../../api/search/search-api-client';
 import {
-  CategoryFacetSection,
-  ConfigurationSection,
-  ContextSection,
-  DateFacetSection,
-  FacetOptionsSection,
-  FacetOrderSection,
-  FacetSection,
-  NumericFacetSection,
-  PaginationSection,
-  ProductListingSection,
-  StructuredSortSection,
+    CategoryFacetSection,
+    ConfigurationSection,
+    ContextSection,
+    DateFacetSection,
+    FacetOptionsSection,
+    FacetOrderSection,
+    FacetSection,
+    NumericFacetSection,
+    PaginationSection,
+    ProductListingSection, ProductListingV2Section,
```

Comment:
Seems like there's some weird code formatter issue ? 🤔

Prettier formatted it like that ?

Reply:
Cleaned this up in https://github.com/coveo/ui-kit/pull/3116/commits/f53124a1c1fd10b30e523e2b5535ca5716e59c94